### DISCUSSION TITLE
Normalization algorithm

### DISCUSSION THREAD

**Discussion by Apirog9** [vdemichev/DiaNN#165] [Q&A] [answered]
Hello, 
What are the details of different normalization methods used in DIA-NN? I am working with peptidomic data with extreme level of matrix interference, so it is very important for me to know how the normalization works in this case, it seem to be quite tricky to achieve good quality.

**Comment by vdemichev** [comment]
Hi, added a section about normalisation in FAQ: https://github.com/vdemichev/DiaNN/blob/master/README.md#frequently-asked-questions
Hope this helps!

**Comment by Apirog9** [comment]
thanks, so I understand the global normalization is done using sum of intensities of all identified peptides in sample?

**Reply by vdemichev** [reply]
No, it's more complex, please see the DIA-NN Nature Methods paper

**Comment by Apirog9** [comment]
Thanks! I have read the paper and I wonder if the value of "p" (in pN precursor fraction used for normalization) could be adjusted for different scenarios. In one case, I got results with severely different median and quantiles after normalization.

**Comment by vdemichev** [comment, marked as answer]
Yes, with --norm-fraction [p], e.g. --norm-fraction 0.3. But it's known that 0.4 works well, cannot guarantee performance if this value is changed. If the end result is not good, you can also easily correct it in R/Python, e.g. on the protein level.
